### PR TITLE
Fix linting errors

### PR DESCRIPTION
**Pull request by lancesnider**
Package: App Project

Describe your changes:

Cleaning up linting errors. 

There's one error left, but I wanted to double check before I fix it. In `CreateCollection.js` there's a non-camelCase identifier named `display_name`. It's used in several places so I wanted to double check that it wasn't on purpose before renaming it. 

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/app-project/src/screens/ClassifyPage/components/CollectionsModal/CollectionsModalContainer.js [hunk 1]
```diff
@@ -29,7 +34,7 @@ class CollectionsModalContainer extends Component {
     this.state = {
       active: false,
       newCollection: {
-        display_name: '',
+        displayName: '',
```
Comment (srallen) [inline packages/app-project/src/screens/ClassifyPage/components/CollectionsModal/CollectionsModalContainer.js]: This change is causing a couple of tests to fail. I think we might want to leave it as `display_name` despite the linter complaining. The resources' attributes are snake case.

Comment (lancesnider) [reply packages/app-project/src/screens/ClassifyPage/components/CollectionsModal/CollectionsModalContainer.js]: Oops! Looks like the linter automatically converted that one. I'll figure out how to make it ignore those errors and not automatically change this one.

Comment (lancesnider) [reply packages/app-project/src/screens/ClassifyPage/components/CollectionsModal/CollectionsModalContainer.js]: @srallen Fixed. I also told `standard` to ignore the lines that were throwing errors: https://github.com/zooniverse/front-end-monorepo/pull/661/files#diff-1277d75a3bea17c9e1e77ad4b9dc4dc3R40

**Comment by srallen** [comment]
I think this branch is having the same problem as we've seen in other repos since we don't allow travis to run on forks.

**Comment by eatyourgreens** [comment]
I'm not sure why Travis isn't running for forked branches, but I've turned off the requirement for branch tests to pass. PR tests still need to pass before merging is allowed.

**Pull request merged**